tests/test_jobcollection.py

```diff
@@ -18,10 +22,86 @@ def test_jobcollection(jobcollection_mock):
     assert len(jobcollection_mock.jobs) == 1
 
 
+def test_jobcollection_multiple(jobcollection_multiple_mock):
+    assert len(jobcollection_multiple_mock.jobs) == 2
+
+
+def test_jobcollection_download_result(jobcollection_mock):
+    with requests_mock.Mocker() as m:
+        download_url = "http://up42.api.com/abcdef"
+        url_download_result = (
+            f"{jobcollection_mock.auth._endpoint()}/projects/"
+            f"{jobcollection_mock.project_id}/jobs/{jobcollection_mock.jobs_id[0]}/downloads/results/"
+        )
+        m.get(url_download_result, json={"data": {"url": download_url}, "error": {}})
+
+        out_tgz = Path(__file__).resolve().parent / "mock_data/result_tif.tgz"
+        out_tgz_file = open(out_tgz, "rb")
```

Comment:
Suggestion: Context manager here would prevent file being left open in memory after test has run

Reply:
Good catch! Changed